Source/WebCore/accessibility/atspi/AccessibilityObjectAtspi.cpp

```diff
@@ -312,7 +312,16 @@ static Atspi::Role atspiRole(AccessibilityRole role)
     case AccessibilityRole::UserInterfaceTooltip:
         return Atspi::Role::ToolTip;
     case AccessibilityRole::WebArea:
+    {
+        RefPtr liveObject = dynamicDowncast<AccessibilityObject>(coreObject);
```

Comment:
this is needed modification of the original patch due to change: https://github.com/WebKit/WebKit/commit/2d8149bbed87335fb0470e35007923445053c4de, frame() is moved from AXCoreObject to AccessibilityObject